### ISSUE TITLE
JAWS does not announce <optgroup> heading.

### ISSUE THREAD

**Issue by Taballe** [FreedomScientific/VFO-standards-support#46]
### Summary

1) Launch JAWS 18.0.4534.

2) Go to this page
https://ibma.github.io/Va11yS/WCAG_HTML/H85_example1.html

 
3) Open the dropdown with ALT+arrow, then use down arrow key to traverse the list.

### Expected result

JAWS announces the <optgroup> headings, thereby helping JAWS user understand the grouping that is obvious to sighted users.

The page being tested shows that this was shown to work in the past.   It also refers to
WCAG 85, which is this:  https://www.w3.org/TR/WCAG20-TECHS/H85.html

### Actual result

JAWS announces the list items (which are reachable via keyboard, so this is normal), but does not announce the optgroup labels/headings (e.g., Fruits, Vegetables, Baked Goods).

 
#### Operating System and version
Windows 10 Pro

####  Browsers
a) Firefox Quantum (57.0.4)
b) Chrome 63.0.3239.132 

####  JAWS Version
18.0.4534

**Comment by Menelion** [comment]
confirmed also with JAWS 2018.1712.10 and Google Chrome Version 66.0.3331.1 Canary.  
It seems though, JAWS never handled `optgroup`s, I had reported this long time ago.

**Comment by stevefaulkner** [comment]
Tested with JAWS 2018 and Firefox 52.6.0 (64-bit) ESR, group labels announced.
Not announced in Google chrome as the group labels are not exposed in the accessibility tree. Note that NVDA does not announce them with Chrome, again because they are not exposed in the acc tree.

@Taballe testing with a) Firefox Quantum (57.0.4) is not currently recommended as it doesn't work well with JAWS. Hopefully this will be resolved in the next few months , until that time suggest using Firefox 52.6.0 (64-bit) ESR

**Comment by stevefaulkner** [comment]
Chrome bug filed: [HTML optgroup elements and their labels not represented in the accessibility tree](https://bugs.chromium.org/p/chromium/issues/detail?id=817579)

**Comment by stevefaulkner** [comment]
closing as bug filed
